Casks/bbedit.rb

```diff
@@ -11,11 +11,15 @@
   end
   url "https://s3.amazonaws.com/BBSW-download/BBEdit_#{version}.dmg",
       verified: "s3.amazonaws.com/BBSW-download/"
-  appcast "https://versioncheck.barebones.com/BBEdit.xml"
   name "BBEdit"
   desc "Text, code, and markup editor"
   homepage "https://www.barebones.com/products/bbedit/"
 
+  livecheck do
+    url "http://versioncheck.barebones.com/BBEdit.xml"
+    regex(%r{<key>SUFeedEntryShortVersionString</key>\s*<string>(\d+(?:\.\d+)+)</string>}i)
```

Comment:
```suggestion
    regex(/BBEdit[._-]v?(\d+(?:\.\d+)+)\.dmg/i)
```

We generally prefer to identify versions from filenames when possible, as this also indicates that an available version is suitable to be used in the formula/cask.